### ISSUE TITLE
clarify 3d geometry type support

### ISSUE THREAD

**Issue by tomkralidis** [geoalchemy/geoalchemy2#157]
Not sure if I'm looking in the right place, but the [docs](https://geoalchemy-2.readthedocs.io/en/latest/types.html#geoalchemy2.types._GISType) do not specify support for 3d geometries, even though the following seems to work:

```python
 location = Column(Geometry('POINTZ', dimension=3, srid=4326))
```

**Comment by elemoine** [comment]
Right. I think the doc needs to be extended.

**Comment by adrien-berchet** [comment]
Fixed in `0.6.3`
